src/main/kotlin/com/example/pass_manager/web/dto/PriceDistribution.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.pass_manager.web.dto
+
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.mapping.Field
+import java.math.BigDecimal
+
+data class PriceDistribution(@Id val typeName: String?, @Field("total") val spentForPassType: BigDecimal,)
```

Comment:
```suggestion
data class PriceDistribution(
    @Id 
    val typeName: String?, 
    @Field("total") 
    val spentForPassType: BigDecimal,
)
```
make it more readable

Reply:
why `typeName` is `@Id`?

Reply:
why mongo model is placed in `.dto` package?
why the model is placed in `.web.` package?

`web` is about web staff and shouldn't depend on DB.
`dto` is about request/response and shouldn't have any relation to the db

Reply:
Done, it might have been dto